src/JSONNestedNode.js

```diff
@@ -147,8 +137,10 @@ export default class JSONNestedNode extends React.Component {
       itemType,
       createItemString(data, collectionLimit)
     );
-
-    const stylingArgs = [keyPath, nodeType, expanded, expandable, this.isShouldExtract()];
+    const hover = maxClickableNodeDepth && keyPath.length > maxClickableNodeDepth
```

Comment:
I think you can just use this.state.hover, because if keyPath.length > maxClickableNodeDepth, then state.hover will never be set.